x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/connector_edit_flyout.tsx

```diff
@@ -32,7 +32,10 @@ import { updateActionConnector, executeAction } from '../../lib/action_connector
 import { hasSaveActionsCapability } from '../../lib/capabilities';
 import { useActionsConnectorsContext } from '../../context/actions_connectors_context';
 import { PLUGIN } from '../../constants/plugin';
-import { ActionTypeExecutorResult } from '../../../../../actions/common';
+import {
```

Comment:
Do the changes to execute, including the following files, belong in this PR?  Seems like they should be in a separate one. 

- `x-pack/plugins/actions/common/types.ts` 
- `x-pack/plugins/triggers_actions_ui/public/application/lib/action_connector_api.ts`,

Reply:
They belong here in that without them it becomes impossible to correctly display the error in this PR.
It *is* addressing a broader problem that I identified while working on this. 🤷

Reply:
Gotcha, thx!